api/app/controllers/spree/api/orders_controller.rb

```diff
@@ -14,6 +14,12 @@ class OrdersController < Spree::Api::BaseController
         end
       end
 
+      def approve
+        authorize! :update, @order, params[:token]
```

Comment:
@jhawthorn thanks for the response!
I'm not sure, but the way to do this is `authorize! :admin, @order, params[:token]` right?